crates/observe/Cargo.toml

```diff
@@ -6,7 +6,7 @@ edition = "2021"
 license = "MIT OR Apache-2.0"
 
 [dependencies]
-atty = "0.2"
+atty = "0.2.14"
```

Comment:
https://rustsec.org/advisories/RUSTSEC-2021-0145 (still not solved with this update, they are working on it)